charts/fluent-bit/templates/configmap.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
   labels:
     {{- include "fluent-bit.labels" . | nindent 4 }}
 data:
+{{ range $key, $value := .Values.extraLuaScripts }}
+  script-{{ $key }}: {{ $value | quote }}
+{{ end }}
```

Comment:
Would it make sense to create a `lua-configmap.yaml` exclusively for lua scripts rather than adding them to the config one?

Reply:
moved to a separate configmap and added sha256 sum of the one into the pod annotations